sky/utils/policy_utils.py

```diff
@@ -0,0 +1,162 @@
+"""Customize policy by users."""
+import copy
+import importlib
+import os
+import tempfile
+import typing
+from typing import Literal, Optional, Tuple, Union
+
+import colorama
+
+from sky import dag as dag_lib
+from sky import exceptions
+from sky import policy as policy_lib
+from sky import sky_logging
+from sky import skypilot_config
+from sky import task as task_lib
+from sky.utils import common_utils
+from sky.utils import ux_utils
+
+logger = sky_logging.init_logger(__name__)
+
+
+def _get_policy_cls(policy: Optional[str]) -> Optional[policy_lib.AdminPolicy]:
+    """Gets admin-defined policy."""
+    if policy is None:
+        return None
+    try:
+        module_path, class_name = policy.rsplit('.', 1)
+        module = importlib.import_module(module_path)
+    except ImportError as e:
+        with ux_utils.print_exception_no_traceback():
+            raise ImportError(
+                f'Failed to import policy module: {policy}. '
+                'Please check if the module is installed in your Python '
+                'environment.') from e
+
+    try:
+        policy_cls = getattr(module, class_name)
+    except AttributeError as e:
+        with ux_utils.print_exception_no_traceback():
+            raise AttributeError(
+                f'Could not find {class_name} class in module {module_path}. '
+                'Please check with your policy admin for details.') from e
+
+    # Check if the module implements the AdminPolicy interface.
+    if not issubclass(policy_cls, policy_lib.AdminPolicy):
+        with ux_utils.print_exception_no_traceback():
+            raise ValueError(
+                f'Policy module {policy} does not implement the AdminPolicy '
```

Comment:
```suggestion
                f'Policy class {policy_cls} does not implement the AdminPolicy '
```

Reply:
Let's use `policy` instead, since `policy_cls` will print some python's internal representation for types